### DISCUSSION TITLE
URL to get specific version and namespace similar to https://strimzi.io/install/latest?namespace=kafka

### DISCUSSION THREAD

**Discussion by pppura** [strimzi/strimzi-kafka-operator#10932] [Q&A]
Hello!

In the quickstart guide this url is provided to get the latest strimzi version: https://strimzi.io/install/latest?namespace=kafka

I wasn't able to find in any documentation a way to use the same API/URL to get a specific version of Strimzi. Is there a way to do this?

**Comment by scholzj** [comment]
No, there is no way to do this. The URL can be used only to install the latest version.
